src/components/SelectionList/TransactionListItem.tsx

```diff
@@ -0,0 +1,310 @@
+            keyForList={item.keyForList}
+            onFocus={onFocus}
+            shouldSyncFocus={shouldSyncFocus}
+            hoverStyle={item.isSelected && styles.activeComponentBG}
+        >
+            {(hovered) => (
+                <>
+                    {canSelectMultiple && (
+                        <PressableWithFeedback
+                            accessibilityLabel={item.text ?? ''}
+                            role={CONST.ROLE.BUTTON}
+                            // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+                            disabled={isDisabled || item.isDisabledCheckbox}
+                            onPress={handleCheckboxPress}
+                            style={[styles.cursorUnset, StyleUtils.getCheckboxPressableStyle(), item.isDisabledCheckbox && styles.cursorDisabled, styles.mr3]}
+                        >
+                            <View style={[StyleUtils.getCheckboxContainerStyle(20), StyleUtils.getMultiselectListStyles(!!item.isSelected, !!item.isDisabled)]}>
+                                {item.isSelected && (
+                                    <Icon
+                                        src={Expensicons.Checkmark}
+                                        fill={theme.textLight}
+                                        height={14}
+                                        width={14}
+                                    />
+                                )}
+                            </View>
+                        </PressableWithFeedback>
+                    )}
+                    <View style={[styles.flexRow, styles.flex1, styles.gap3]}>
+                        <View style={[styles.flex1, styles.flexColumn, styles.justifyContentCenter, styles.alignItemsStretch]}>{dateElement}</View>
+                        <View style={[styles.flex1, styles.flexColumn, styles.justifyContentCenter, styles.alignItemsStretch]}>{descriptionElement}</View>
+                        <View style={[styles.flex1, styles.flexColumn, styles.justifyContentCenter, styles.alignItemsStretch]}>
+                            <View style={[styles.flexRow, styles.gap3, styles.flex1, styles.alignItemsCenter]}>
+                                <Avatar
+                                    imageStyles={[styles.alignSelfCenter]}
+                                    size={CONST.AVATAR_SIZE.SMALL}
+                                    source={personalDetails[item.managerID]?.avatar}
+                                    name={personalDetails[item.managerID]?.displayName}
+                                    type={CONST.ICON_TYPE_WORKSPACE}
+                                />
+                                <Text
+                                    numberOfLines={1}
+                                    style={[styles.flex1, styles.flexGrow1, styles.textStrong]}
+                                >
+                                    {personalDetails[item.managerID]?.displayName}
```

Comment:
`From` should be `item.accountID`
```suggestion
                                    {personalDetails[item.accountID]?.displayName}
```